### DISCUSSION TITLE
Extra output on `pscale auth login` command

### DISCUSSION THREAD

**Discussion by sverch** [planetscale/discussion#37] [Bug Reports] [answered]
When I run `pscale auth login`, it runs normally (and I can complete the login flow because it opens in a browser), but it starts spamming a bunch of garbage characters in my terminal that overwrites everything with `WWWWWWWWWWWWWWWWWWWWWWWWWWWWWW`.

```
$ uname -a
Linux transition 5.4.0-72-generic #80-Ubuntu SMP Mon Apr 12 17:35:00 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```
Here's the log with all the extra characters: [screenlog.txt](https://github.com/planetscale/beta/files/6528419/screenlog.txt).

**Comment by sverch** [comment]
Actually, I'm having this problem with other commands that have a wait message as well.

**Comment by aquarapid** [comment, marked as answer]
Yeah, we've been chasing some interactions with various terminals and the ANSI escape codes code in the `pscale` cli.  What do you have for your `TERM` variable;  and are you using anything special in the terminal (e.g. `screen` or `tmux`, etc.)?

**Reply by sverch** [reply]
```
$ echo $TERM
xterm-color
```

I just used screen to capture all the raw escape codes for sharing here, but it happened in my normal terminal.

**Reply by aquarapid** [reply]
Interesting.  I'm using an ubuntu docker image to try and reproduce without luck (although I do see the/a problem inside `screen`).  What terminal emulator are you using?

**Reply by sverch** [reply]
```
$ urxvt -help 2>&1 | grep -i released
rxvt-unicode (urxvt) v9.22 - released: 2016-01-23
```

**Reply by aquarapid** [reply]
Confirmed;  I can reproduce locally (non-Ubuntu) with `urxvt v9.22` as well.  `TERM` does not seem to matter much, happens with `xterm-color` or `vt100`.
